Sources/StreamChatUI/CommonViews/Attachments/AttachmentPlaceholder.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// Default provider that is used when AttachmentPreviewProvider is not implemented for custom attachment payload.
+public struct AttachmentPreviewPlaceholderProvider: AttachmentPreviewProvider {
+    public func previewView<ExtraData: ExtraDataTypes>(components: _Components<ExtraData>) -> UIView {
+        AttachmentPlaceholderView().withoutAutoresizingMaskConstraints
+    }
+}
```

Comment:
nit: I find that naming actual "Default" on default implementations it is easier to understand what is this implementation about
```suggestion
/// Default provider that is used when AttachmentPreviewProvider is not implemented for custom attachment payload.
public struct DefaultAttachmentPreviewProvider: AttachmentPreviewProvider {
    public func previewView<ExtraData: ExtraDataTypes>(components: _Components<ExtraData>) -> UIView {
        AttachmentPlaceholderView().withoutAutoresizingMaskConstraints
    }
}
```